APIs/User.md

```
@@ -0,0 +1,105 @@
+POST: `/api/User/profile/{followerId}/unfollow/{targetUserId}`
+
+**Response**
+
+Status Code: 200
+
+---------
+### Update Password
+GET: `/api/User/profile/update/pass/{userId}`
+
+**Request**
+```json
```

markdownlint-cli2 (0.17.2)

Horizontal rule style
Expected: -----; Actual: ------

(MD035, hr-style)